File: src/main/kotlin/it/pagopa/wallet/eventdispatcher/warmup/utils/EventsUtil.java
package it.pagopa.wallet.eventdispatcher.warmup.utils;

import it.pagopa.wallet.eventdispatcher.common.cdc.WalletDeletedEvent;
import it.pagopa.wallet.eventdispatcher.domain.WalletCreatedEvent;

import java.time.OffsetDateTime;

public class EventsUtil {
    public static WalletCreatedEvent getWalletCreatedEvent() {
        return new WalletCreatedEvent(
                "00000000-0000-0000-0000-000000000001",
                OffsetDateTime.now(),
                "00000000-0000-0000-0000-000000000000"
        );
    }

    public static WalletDeletedEvent getWalletDeleteLoggingEvent() {
        return new WalletDeletedEvent(
                "00000000-0000-0000-0000-000000000001",
                "2025-01-10T14:28:47.843515440Z[Etc/UTC]",
                WalletDeletedEvent.class.getSimpleName(),
                "00000000-0000-0000-0000-000000000000"
        );
    }

}

File: src/main/kotlin/it/pagopa/wallet/eventdispatcher/warmup/utils/EventsUtil.kt
package it.pagopa.wallet.eventdispatcher.warmup.utils

import it.pagopa.wallet.eventdispatcher.common.cdc.WarmupLoggingEvent
import it.pagopa.wallet.eventdispatcher.domain.WalletCreatedEvent
import java.time.OffsetDateTime

object EventsUtil {
    fun getWalletCreatedEvent(): WalletCreatedEvent {
        return WalletCreatedEvent(
            eventId = "00000000-0000-0000-0000-000000000001",
            creationDate = OffsetDateTime.now(),
            walletId = "00000000-0000-0000-0000-000000000000"
        )
    }

    fun getWarmupLoggingEvent(): WarmupLoggingEvent {
        return WarmupLoggingEvent()
    }
}
